scripts/update

```diff
@@ -61,26 +62,21 @@ if [[ "${BASH_SOURCE[0]}" == "${0}" ]]; then
   if [[ "${1:-}" == "--help" ]]; then
     usage
   else
-    # Get the latest copy of the gamemode
-    if [[ ! -d src/addons/zombiesurvival ]]; then
-      git clone https://github.com/JetBoom/zombiesurvival.git src/addons/zombiesurvival
-    else
-      pushd src/addons/zombiesurvival
-      git pull
-      popd
-    fi
-
```

Comment:
I followed Atlassian's [subtree blog](https://www.atlassian.com/git/tutorials/git-subtree) to incorporate our own fork into the repository in https://github.com/rbreslow/zs/commit/cb6206b43c8ea4896305cf26979de75a837fce23.